appholder/src/main/java/com/android/mdl/app/document/DocumentDatabase.kt

```diff
@@ -42,6 +42,10 @@ abstract class DocumentDatabase : RoomDatabase() {
             }
         }
 
+        // TODO: According to https://developer.android.com/guide/topics/data/autobackup this
```

Comment:
File an issue on GitHub for this?

Reply:
Sure. Filed as https://github.com/google/identity-credential/issues/102